<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LFileWidget</class>
 <widget class="QWidget" name="LFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="yes">
     <property name="text">
      <string>--&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeView" name="fileTree"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="cunrrentPath"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnPrev">
     <property name="text">
      <string>&lt;--</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
